data.sql

```diff
@@ -18,5 +18,54 @@ VALUES ('Charmander','2020-02-08', 0, false,  11),
 ('Ditto','2022-05-14', 4, true,  22);
 
 
+INSERT INTO owners (full_name, age)
+VALUES ('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
 
+INSERT INTO species(name)
+VALUES ('Pokemon'),
+('Digimon');
 
+-- updating animals foreign keys
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 1 WHERE species_id IS NULL;
+
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name IN('Gabumon','Pikachu');
+UPDATE animals SET owner_id = 3 WHERE name IN('Devimon','Plantmon');
+UPDATE animals SET owner_id = 4 WHERE name IN('Charmander','Squirtle','Blossom');
+UPDATE animals SET owner_id = 5 WHERE name IN('Angemon','Boarmon');
+
```

Comment:
- [ ] I noticed that some queries are missing, according to the project requirement, you need to insert the following data for specialties ⬇️ :
- Vet William Tatcher is specialized in Pokemon.
- Vet Stephanie Mendez is specialized in Digimon and Pokemon.
- Vet Jack Harkness is specialized in Digimon.
Please insert them in order to have the correct output for your queries also don't forget to add screenshot with correct output 👍